src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
+  const multiplier = () => {
+    return days * 40;
+  };
```

Comment:
use variable in such cases, not function

```suggestion
  const total = days * 40;
```